include/ceed/jit-source/cuda/cuda-ref-basis-nontensor-templates.h

```diff
@@ -10,8 +10,6 @@
 #ifndef CEED_CUDA_REF_BASIS_NONTENSOR_TEMPLATES_H
 #define CEED_CUDA_REF_BASIS_NONTENSOR_TEMPLATES_H
 
-#include <ceed.h>
```

Comment:
We have historically included `#include <ceed.h>` in these files so IDEs can more easily understand types in the file.

Reply:
Hmm, let me double check but it should be not actually part of the JiTed string

Reply:
Oh, maybe I did not look closely enough. From [ceed-jit-tools.c](https://github.com/CEED/libCEED/blob/main/interface/ceed-jit-tools.c#L131), maybe system headers which are incuded with `<*>` are ignored, unless they match one of the four given specific ones?

Reply:
```

      bool  is_ceed_header    = is_hash_include && next_left_chevron && (next_new_line - next_left_chevron > 0) &&
                            (!strncmp(next_left_chevron, "<ceed/jit-source/", 17) || !strncmp(next_left_chevron, "<ceed/types.h>", 14) ||
                             !strncmp(next_left_chevron, "<ceed/ceed-f32.h>", 17) || !strncmp(next_left_chevron, "<ceed/ceed-f64.h>", 17));
```

Yeah, it should only be these 4 headers that are dropping in for JiT, so including `#include <ceed.h>` should result in the line being omitted from the source string for JiT.

Reply:
Cool, reverted in [d69fde3](https://github.com/CEED/libCEED/pull/1450/commits/d69fde354f92f0f87cc1cc99880249b8250190ee) and addressed [this comment](https://github.com/CEED/libCEED/pull/1450#discussion_r1463925390) in [0854542](https://github.com/CEED/libCEED/pull/1450/commits/085454240ef0787e2bf0778613c6c2e31bdf9a43).